pommerman/agents/player_agent_blocking.py

```diff
@@ -0,0 +1,37 @@
+from time import time
+import click
```

Comment:
I am unfamiliar with this package. Is it https://click.palletsprojects.com/en/7.x/? 
If so, that page suggests it's for CLIs. Why not do as we did here with Pyglet --> https://github.com/MultiAgentLearning/playground/blob/master/pommerman/agents/player_agent.py

Reply:
Yes, that click.
I just wanted simple code, I found it simpler this way.
I understand if you dont like this dependency, Im sure there are other ways.

Reply:
It does cli things but also some terminal handling (like getchar)